enhancements/api-review/azure_user_defined_tags.md

```diff
@@ -0,0 +1,322 @@
+---
+title: azure_user_defined_tags
+authors:
+  - "@bhb"
+reviewers:
+  - "@patrickdillon"
+  - "@sdodson"
+  - "@jhixson74"
+  - "@JoelSpeed"
+  - "@dhellmann"
+  - "@jewzaam"
+  - "@adambkaplan"
+  - "@abhat"
+  - "@dmage"
+  - "@Miciah"
+  - "@abutcher"
+  - "@staebler"
+  - "@jerpeter1"
+approvers:
+  - "@sdodson"
+  - "@jerpeter1"
+  - "@bparees"
+api-approvers:
+  - "@joelspeed"
+  - "@bparees"
+creation-date: 2022-07-12
+last-updated: 2022-07-12
+tracking-link:
+  - https://issues.redhat.com/browse/OCPPLAN-8155
+  - https://issues.redhat.com/browse/CORS-2249
+see-also:
+  - "enhancements/api-review/custom-tags-aws.md"
+replaces:
+  - N/A
+superseded-by:
+  - N/A
+---
+
+# Apply user defined tags to all Azure resources created by OpenShift
+
+## Summary
+
+This enhancement describes the proposal to allow an administrator of Openshift to 
+have the ability to apply user defined tags to those resources created by Openshift 
+in Azure.
+
+## Motivation
+
+Motivations include but are not limited to:
+
+- Allow admin, compliance, and security teams to keep track of assets and objects 
+  created by OpenShift in Azure.
+
+### User Stories
+
+- As an openshift administrator, I want to have tags added to all resources created 
+  in Azure by Openshift.
+- As an openshift administrator, I want to restrict access granted to Openshift 
+  specific account.
+
+### Goals
+
+- The administrator or service (in the case of Managed OpenShift) installing OpenShift 
+  can configure allowed number of user-defined tags in the Openshift installer generated
+  install config, which is referred and applied by the installer and the in-cluster operators
+  on the the Azure resources during cluster creation.
+- Tags must be applied at creation time, in an atomic operation. It isn't acceptable 
+  to create an object and to apply tags post cluster creation.
+
+### Non-Goals
+
+- Management(update/delete) of resource tags post creation of cluster is out of scope.
+
+## Proposal
+
+A tag of the form `kubernetes.io/cluster/<cluster_name>:owned` will be added to every
```

Comment:
is that whole thing the tag name, with a tag value of `true`?  Or is `owned` the tag value?

Reply:
also why kubernetes and not openshift?  none of this is upstream behavior, right?

what did we name the tag on AWS?

Reply:
`kubernetes.io/cluster/<cluster_name>` is the tag key and `owned` is the tag value.

I am not sure why `kubernetes` was used in the tag key, but when I checked the code, it's used for other cloud platform resources too like AWS, GCP, IBMCloud, AlibabaCloud.

Reply:
it would be good to mention that this tag name is the same tag name we use for other clouds